packages/models-library/src/models_library/api_schemas_datcore_adapter/datasets.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from enum import Enum, unique
+from pathlib import Path
+
+from pydantic import BaseModel, ByteSize
+
+
+class DatasetMetaData(BaseModel):
+    id: str
```

Comment:
TIP: `id: IDStr`

Reply:
I have no control over that string and don't want to open a can of worms.